addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -520,6 +527,13 @@ def add_extra_pickings(self, picking_type_id):
             raise ValidationError(_("Can only add work to ephemeral batches"))
 
         pickings.write({'batch_id': self.id})
+        PickingType = self.env['stock.picking.type']
```

Comment:
We usually have this at the start of the function.

Reply:
Removed.